<ui version="4.0" >
 <class>WExecutionManipulationUI</class>
 <widget class="QDialog" name="WExecutionManipulationUI" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>422</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <widget class="QLabel" name="source_label_p" >
     <property name="text" >
      <string>Source:</string>
     </property>
     <property name="buddy" >
      <cstring>source_p</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QLineEdit" name="source_p" />
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="destination_label_p" >
     <property name="text" >
      <string>Destination:</string>
     </property>
     <property name="buddy" >
      <cstring>destination_p</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="QLineEdit" name="destination_p" />
   </item>
   <item row="2" column="0" colspan="2" >
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Preview</string>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" >
       <widget class="QTableWidget" name="preview_p" >
        <property name="editTriggers" >
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="selectionMode" >
         <enum>QAbstractItemView::NoSelection</enum>
        </property>
        <property name="textElideMode" >
         <enum>Qt::ElideLeft</enum>
        </property>
        <property name="showGrid" >
         <bool>true</bool>
        </property>
        <property name="gridStyle" >
         <enum>Qt::DotLine</enum>
        </property>
        <property name="sortingEnabled" >
         <bool>false</bool>
        </property>
        <property name="wordWrap" >
         <bool>true</bool>
        </property>
        <property name="cornerButtonEnabled" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" colspan="2" >
    <widget class="QLabel" name="message_p" >
     <property name="font" >
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text" >
      <string>Message</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2" >
    <widget class="QDialogButtonBox" name="buttons_p" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttons_p</sender>
   <signal>accepted()</signal>
   <receiver>WExecutionManipulationUI</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttons_p</sender>
   <signal>rejected()</signal>
   <receiver>WExecutionManipulationUI</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
